mybinder/values.yaml

```
@@ -10,6 +10,20 @@ cryptnono:
       containerdHostPath: /run/containerd/containerd.sock
       dockerHostPath: /run/dind/docker.sock
 
+registry:
+  enabled: false
+  config:
+    version: 0.1
+    auth:
+      htpasswd:
+        path: /etc/distribution/auth.htpasswd
+  service:
+    type: ClusterIP
+  storage:
+    filesystem:
+      storageClassName: ""
+      size: 10Gi
```

```suggestion
      # Size is currently ignored- using local path provisioner
      size: 10Gi
```
I presume you're just using K3s' default local storage?